packages/compass-aggregations/src/components/saved-pipelines/saved-pipeline-card.tsx

```diff
@@ -123,7 +116,44 @@ const SavePipelineCard: React.FunctionComponent<SavePipelineCardProps> = ({
   );
 };
 
-export default connect(null, {
-  onOpenPipelineConfirm: openPipelineById,
-  onDeletePipeline: deletePipeline
-})(SavePipelineCard);
+export default connect(
+  (state: RootState) => {
+    return {
+      editor_view_type: mapPipelineModeToEditorViewType(
+        state.pipelineBuilder.pipelineMode
+      ),
+    };
+  },
+  {
+    onOpenPipelineConfirm: openPipelineById,
+    onDeletePipeline: deletePipeline,
+  },
+  (
+    mapProps,
+    dispatchProps,
+    ownProps: Pick<SavePipelineCardProps, 'id' | 'name'>
+  ): SavePipelineCardProps => {
```

Comment:
Doesn't look like this is needed, redux types should be able to infer this correctly on their own (similar to other places where you use this third prop)